src/gh_pr/auth/token.py

```
@@ -177,11 +219,15 @@ def get_token_info(self) -> Optional[dict[str, Any]]:
             info["scopes"] = []  # Unable to determine scopes without private attribute access
 
             # Check token expiration for fine-grained tokens
-            if token_type == "Fine-grained Personal Access Token":
-                # Fine-grained tokens have expiration
-                # GitHub API doesn't directly expose token expiry
-                # In real implementation, you might store this when token is created
-                pass
+            if token_type == "Fine-grained Personal Access Token" and self.config_manager:
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "token_type"

(S105)